.github/workflows/ci.yml

```
@@ -0,0 +1,77 @@
+name: CI - Ansible Collection Quality Checks
```

Let's make this concise:

```suggestion
name: CI
```